src/local.h

```diff
@@ -1171,13 +1176,14 @@ MDSP_BOOL gtoj(int, int, int, int *, int *);
 MDSP_BOOL ccap2std(char *, Devaddr *, Diskaddr *);
  
 MDSP_BOOL std2ccap(Devaddr *, Diskaddr *, char *);
- 
+
+#ifndef _WIN32
 char *strupr(char *);
 char *strlwr(char *);
 /* char *strdup(char *); */
 //char *strndup(char *, int);
 int strcmpi(char *, char *);
```

Comment:
This strcmpi can be deleted or commented out. It is not actually referenced anywhere in the code.

Reply:
Removed several unneeded prototypes